### PR TITLE
update info copy to 2018

### PR DESCRIPTION
**Pull request by Kriesse**
still more work to do (add links…), but it should be safe to share now. removed all misleading / outdated 2017 info.

### CODE REVIEW

File: contents/copy/info/main.md [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
+CSSconf EU will take place on June 1st, 2018 in Berlin’s Arena Halle, offering a day packed with inspiring and insightful talks and time well spent in the company of great people. This year we’re planning the most adventurous and ambitious CSSconf EU ever – be prepared for CSSconf EU, the Festival Edition, featuring a newer, bigger, more exciting venue that we’ll pack with some surprises.
 
 ### Speakers, Topics, Language
 
 CSSconf EU unites top international speakers and newcomers with fresh ideas. They will discuss cutting-edge technologies and questions around front-end development and web design – with a specific focus on CSS.
 
 See full line-up and Schedule
 
-Talks are always in English, and video recordings will be released for free after the event. All speakers were selected via an open Call for Speakers. The voting on all proposals took place in January 2017.
+Talks are always in English, and video recordings will be released for free after the event. All speakers re selected via an open Call for Speakers. The voting on all proposals takes place in January 2018.
```
Comment (Lucie-H) [inline contents/copy/info/main.md]: `All speakers re selected` > All speakers are selected

File: contents/copy/info/main.md [hunk 2]
```diff
@@ -40,22 +40,26 @@ Eichenstraße 4, Berlin
 
 This year, for the first time ever, we have a Festival Area! We dedicated a part of our large and beautiful venue just to fun activities and short live demos that you can check out during breaks. You’ll be able to: meet the companies that support CSSconf EU, chat with the people behind open source projects / community initiatives, be wooed by { live : JS }, see the CSSconf bags getting printed, take selfies in our photo booth, play foosball, charge up your phones, grab a frozen yogurt and … sit on beanbags!
```
Comment (Lucie-H) [inline contents/copy/info/main.md]: from here on everything looks like in italics, is that just github?

File: contents/copy/info/main.md (hunk 1 above)
Comment (Kriesse) [reply contents/copy/info/main.md]: fixed

File: contents/copy/info/main.md (hunk 2 above)
Comment (Kriesse) [reply contents/copy/info/main.md]: yeah, not sure why :/ it looks correct tho when rendered.

**Pull request merged**